src/contribution-collector.js

```diff
@@ -53,79 +59,84 @@ export async function run() {
  * - Collects user data from WordPress.org.
  * - Generates a list of contributors.
  *
- * @returns {Promise<string>}
+ * @return {Promise<string>} null if no contributors, otherwise an object with lists of contributors.
```

Comment:
Should this be reversed? It doesn't look like it's explicitly stated in the [JS standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/#grammar), but I know with PHP the [default value is supposed to be stated last](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods).

Reply:
I updated it